CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.2.0
+
+- Allow any version of numpy
```

Comment:
https://github.com/lace/tri-again/compare/1.1.2...main

The numpy requirement does not seem to have changed. But what has changed is that we require a polliwog 3.x prerelease. I think this is a breaking change, and should be shipped as 2.0.0a0 (and bumped to 2.0.0 when we release polliwog 3.0).

Reply:
ah OK.  Makes sense.